learn/installing-ballerina.md

```diff
@@ -17,44 +17,29 @@ redirect_from:
 
 # Installing Ballerina
 
-- [Updating Ballerina](#updating-ballerina)
 - [Installing Ballerina via installers](#installing-ballerina-via-installers)
 - [Installing via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file)
+- [Updating Ballerina](#updating-ballerina)
 - [Building from source](#building-from-source)
 - [Uninstalling Ballerina](#uninstalling-ballerina)
 - [Getting help](#getting-help)
 - [What is next](#what-is-next)
 
-## Updating Ballerina
-
-If you already have a jBallerina version installed, you can use the update tool (by executing the below commands) to update to the latest jBallerina version as follows. 
-
->**Note:** If you are a new user who has not installed jBallerina, install via the [installers](#installing-ballerina-via-installers).
-
-**The jBallerina Version**|**Update Tool Usage**
-:-----:|:-----:
-jBallerina version 1.2.0 or above|`ballerina dist update`
-jBallerina 1.2.0 but switched to a previous version|`ballerina dist pull jballerina-<LATEST-VERSION>`
-jBallerina version 1.1.0 or below|Use the [installers](#installing-ballerina-via-installers)
-
-
 ## Installing Ballerina via installers
 
-Ballerina installers support the below operating systems:
+Follow the below instructions to download and use the installers to install the latest Ballerina version. The installer will automatically uninstall the old Ballerina version if you have one already installed.
 
-- Windows 7 or later
-- Ubuntu Linux 
-- Red Hat Enterprise Linux
-- Cent OS
-- Mac OS 
+>**Info:** Ballerina installers support operating systems such as Windows, Ubuntu Linux, Red Hat Enterprise Linux, Maxc OS, and Cent OS. If you are using an unsupported operating system, [install via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file).
 
->**Info:** If you are using an unsupported operating system, [install via the Ballerina language ZIP file](#installing-via-the-ballerina-language-zip-file).
+### Installing on MacOS
 
-Follow the below instructions to download and use the installers to install the latest Ballerina version. The installer will automatically uninstall the old Ballerina version if you have one already installed.
+[Download the package file](/downloads) and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution in the `/Library/Ballerina` directory.
 
-### Installing on Mac OS
+> **Info:** Alternatively, you can execute the below command to install Ballerina in MacOS using Homebrew.
```

Comment:
The proper way to use below is after the noun so shall we edit it to:

Alternatively, you can execute the command below to install Ballerina in macOS using Homebrew.

If we have done this in other places, let's try to fix those as well

Reply:
Fixed.